docs/install/Knative-with-MicroK8s.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Install on MicroK8s"
+linkTitle: "MicroK8s"
+weight: 10
+type: "docs"
+---
+
+This guide walks you through the installation of Knative using [MicroK8s](https://microk8s.io).
+
```

Comment:
Can you add a bit more background on microk8s ? What it is, what OS it runs on etc...